AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -232,8 +232,8 @@ NS_ASSUME_NONNULL_BEGIN
 /** @name Observing node state changes */
 
 /**
-  * Declare <ASInterfaceState> methods as requiring super calls (this can't be required in the protocol).
-  * For descriptions, see <ASInterfaceState> definition.
+  * Declare <ASInterfaceStateDelegate> methods as requiring super calls (this can't be required in the protocol).
```

Comment:
This was a spot that hadn't been renamed after there was a request to put Delegate in the name.